11ty/definitions/suicide.md

```diff
@@ -5,4 +5,25 @@ defined: false
 flag:
   level: warning
   text: content warning
+reading:
+  - text: "Mental Health Foundation: Suicide"
+  href: https://www.mentalhealth.org.uk/a-to-z/s/suicide
+  - text: "Samaritans: Best Practice Suicide Reporting Tips"
+  href: https://www.samaritans.org/about-samaritans/media-guidelines/best-practice-suicide-reporting-tips/
 ---
+the deliberate act of voluntarily ending one's own life. Motivation for suicide can be the result of conscious or unconscious thoughts and feelings.
+
+## Issues
```

Comment:
I would rename this to ## Usage Notes, as the word suicide itself isn't the issue, but rather its incorrect usage (it's a bit pedantic, but I think important to articulate the nuance here).